qtdialogs/qtdialogs.py

```diff
@@ -390,17 +392,12 @@ def __init__(self, parent=None, main=None, initLabel=''):
       self.btnbox.addButton(self.btnCancel, QtWidgets.QDialogButtonBox.RejectRole)
       self.btnbox.addButton(self.btnAccept, QtWidgets.QDialogButtonBox.AcceptRole)
 
-      self.connect(self.btnAdvCrypto, SIGNAL('toggled(bool)'), \
-                   self.cryptoFrame, SLOT('setVisible(bool)'))
-      self.connect(self.btnAccept, SIGNAL(CLICKED), \
-                   self.verifyInputsBeforeAccept)
-      self.connect(self.btnCancel, SIGNAL(CLICKED), \
-                   self, SLOT('reject()'))
-
-
       self.btnImportWlt = QtWidgets.QPushButton(self.tr("Import wallet..."))
-      self.connect(self.btnImportWlt, SIGNAL("clicked()"), \
-                    self.importButtonClicked)
+      self.connect(self.btnAdvCrypto, SIGNAL('toggled(bool)'), self.cryptoFrame, SLOT('setVisible(bool)'))
```

Comment:
call the slot directly instead:
`self.btnAdvCrypto.toggled.connect(setVisible)`